crates/core/src/db/datastore/locking_tx_datastore/mut_tx.rs

```diff
@@ -692,6 +692,23 @@ impl MutTxId {
     pub fn rollback(self) {
         // TODO: Check that no sequences exceed their allocation after the rollback.
     }
+
+    pub fn get_row_count(&self, table_id: TableId) -> RowCount {
```

Comment:
Same here. I'm fine if you want to remove this for now and then add it back when we start using cardinality estimation for DML.